toml/src/test/kotlin/org/rust/toml/inspections/CargoTomlUnresolvedPathReferenceInspectionTest.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.inspections
+
+import org.rust.ide.inspections.RsInspectionsTestBase
+import org.toml.ide.inspections.TomlUnresolvedReferenceInspection
+
+class CargoTomlUnresolvedPathReferenceInspectionTest : RsInspectionsTestBase(TomlUnresolvedReferenceInspection::class) {
+    fun `test build script found`() = checkByFileTree("""
+        //- main.rs
+        fn main() {}
+
+        //- foo.rs
+        fn foo() {}
+
+        //- Cargo.toml
+        [package]
+        name = "example"
+        build = "foo.rs"/*caret*/
+    """)
+
+    fun `test build script not found`() = checkByFileTree("""
+        //- main.rs
+        fn main() {}
+
+        //- Cargo.toml
+        [package]
+        name = "example"
+        build = "<error descr="Cannot resolve file 'foo.rs'">foo.rs</error>"/*caret*/
```

Comment:
Tests fail because now we use `warning` level for annotation but tests expect `error`

Reply:
Sorry, forgot to run them.